rails/locale/es.yml

```diff
@@ -17,7 +17,7 @@ es:
     - vie
     - sáb
     abbr_month_names:
-    - 
+    -
```

Comment:
My VS Code automatically cleans trailing whitespaces on save. So it created this changes, and I only noticed after the PR was created.

I don't think we need the whitespaces, but let me know if you want me to add the back again.